frontend/src/scenes/dashboard/newDashboardLogic.ts

```diff
@@ -77,6 +78,9 @@ export const newDashboardLogic = kea<newDashboardLogicType>([
             actions.submitNewDashboard()
         },
         showNewDashboardModal: () => {
+            actions.getAllDashboardTemplates()
```

Comment:
don't reset the modal when opening it, or you can't set values on the form when opening it